packages/vscode/package.json

```diff
@@ -50,6 +50,10 @@
         "title": "Glint: Show IR for Debugging",
         "command": "glint.show-debug-ir",
         "enablement": "config.glint.debug == true"
+      },
+      {
+        "title": "Glint: Sort Imports",
+        "command": "glint.sort-imports"
```

Comment:
Can we limit this command to only show up on files where it's relevant? It looks like the equivalent commands from Code's built-in JS/TS language features extension matches on [`editorLangId`](https://code.visualstudio.com/api/references/when-clause-contexts#available-context-keys).

Reply:
Good idea!  I'll set this up. How should the command be named? 
- `Glint: Sort Imports`: no language 
- `Glint: Sort Typescript Imports`: inline language 
- `Typescript: Sort Imports`: prefix language. This mimics the built-in extension. could be helpful or confusing, not sure which

GlimmerJS & GlimmerTS could be languages as well for their respective files.